mne_bids/config.py

```diff
@@ -172,21 +172,26 @@
 
 # mapping from BIDs coordinate frames -> MNE
 BIDS_TO_MNE_FRAMES = {
-    'ctf': 'ctf_head',
-    '4dbti': 'ctf_head',
-    'kityokogawa': 'ctf_head',
-    'elektaneuromag': 'head',
-    'chietiitab': 'head',
-    'captrak': 'head',
-    'acpc': 'ras',
-    'mni': 'mni_tal',
-    'fs': 'fs_tal',
-    'ras': 'ras',
-    'voxel': 'mri_voxels',
-    'mri': 'mri',
-    'unknown': 'unknown'
+    'CTF': 'ctf_head',
+    '4DBti': 'ctf_head',
+    'KitYokogawa': 'ctf_head',
+    'ElektaNeuromag': 'head',
+    'ChietiItab': 'head',
+    'CapTrak': 'head',
+    'ACPC': 'ras',  # XXX: there is no ACPC in mne-python
```

Comment:
should this better become `'unknow'` then?

Reply:
isn't ACPC assumed to be in RAS? Would it be better to try to get the data annotated as much as possible?

Reply:
Ok, I trust your judgment there :)